src/Npgsql/Properties/AssemblyInfo.cs

```diff
@@ -22,6 +22,13 @@
 "7aa16153bcea2ae9a471145624826f60d7c8e71cd025b554a0177bd935a78096" +
 "29f0a7afc778ebb4ad033e1bf512c1a9c6ceea26b077bc46cac93800435e77ee")]
 
+[assembly: InternalsVisibleTo("Npgsql.NodaTime, PublicKey=" +
```

Comment:
This is needed because BCL type handler Ctors/Methods are internal. Since i need to delegate to them, InternalsVisibleTo is needed.

> I don't see any references to internals in NodaTime tests.

Its not for the `Npgsql.PluginTests` assembly, its for `Npgsql.NodaTime´ assembly.

Reply:
I think it's best to make these ctors/methods public rather than introducing IVT.

The type handlers are currently somewhat inconsistent with public/internal - we've made public certain things because we needed them, but not others. As a general rule, the type handers should be a sort of "public internal" (similar to how EF Core manages things): they should be public for easy extension, but without any guarantees against breaking changes (at least in the near future).